services/web/server/tests/integration/01/test_garbage_collection.py

```diff
@@ -344,9 +345,16 @@ async def disconnect_user_from_socketio(
     await sio.disconnect()
     assert not sio.sid
     await asyncio.sleep(0)  # just to ensure there is a context switch
-    assert not await socket_registry.find_keys(("socket_id", sio.get_sid()))
-    assert sid not in await socket_registry.find_resources(resource_key, "socket_id")
-    assert not await socket_registry.find_resources(resource_key, "socket_id")
+
+    async for attempt in AsyncRetrying(
+        wait=wait_fixed(0.1), stop=stop_after_delay(5), reraise=True
+    ):
+        with attempt:
+            assert not await socket_registry.find_keys(("socket_id", sio.get_sid()))
+            assert sid not in await socket_registry.find_resources(
+                resource_key, "socket_id"
+            )
+            assert not await socket_registry.find_resources(resource_key, "socket_id")
```

Comment:
what does this have to do with pydantic v2?
if this is a fix for master this should go there. otherwise merging changes will become a nightmare.

Reply:
OK, reverting. Did not think about that.